projects/components/src/table/table.component.ts

```diff
@@ -358,8 +357,11 @@ export class TableComponent
       this.isTableFullPage = this.display === TableStyle.FullPage;
     }
 
-    if (changes.mode || changes.columnConfigs || changes.detailContent || changes.metadata) {
+    if (changes.mode || changes.detailContent || changes.metadata) {
       this.initializeColumns();
+    } else if (changes.columnConfigs) {
+      !isEqualIgnoreFunctions(changes.columnConfigs.previousValue, changes.columnConfigs.currentValue) &&
```

Comment:
Why is this needed?

Reply:
Which part?

Reply:
we're special casing column configs to not use reference equality - that's generally a responsibility delegated to the component passing in an input.